modules/ROOT/pages/design-modify-raml-specs-conform.adoc

```diff
@@ -0,0 +1,791 @@
+= Modify Published RAML API Specifications to Conform Completely to RAML 0.8 or 1.0
+
+[[bookmark-a,Back to the top]]
+
+As of November 1, 2018, the API editor Design Center is now more precise in ensuring that the RAML in your API specifications is valid. However, it is possible that one or more of the APIs that you used previous versions of the API editor to create and publish to Anypoint Exchange includes invalid RAML that those previous versions did not flag.
+
+Check your published APIs by opening them in the API editor in Design Center. You can use the guide on this page to correct a number of errors that MuleSoft has found while testing the current API editor on a large number of RAML API specifications. If you see errors that this guide does not include, refer to the specifications for RAML 0.8 and 1.0, which you can read on https://raml.org/[raml.org].
+
+You have until November 1, 2019 to correct any invalid RAML that your published API specifications might include. After that date, you will be unable to republish and redeploy APIs in which there is invalid RAML.
```

Comment:
@chiragnarangmulesoft and I talked about this date being more clear about this not being a typo.

```suggestion
You have until October 31, 2019 to correct any invalid RAML that your published API specifications might include. After that date, you will be unable to republish and redeploy APIs in which there is invalid RAML.
```